### ISSUE TITLE
Cannot install any extensions.

### ISSUE THREAD

**Issue by Neutrino-Sunset** [Microsoft/vscode#25228]
- VSCode Version: 1.11.2
- OS Version: Win7 x64

Steps to Reproduce:
1. Install VS Code.
2. Launch VS Code.
3. Press CTRL+SHIFT+X to open extensions view.
4. Select 'Show Recommended Extensions', (or seach for an extension).

Result:
A blue bar scrolls from left to right 5 times, then stops.

Expected:
To see a list of extensions to install.

Description:
New install of VSCode, on a workstation that does not connect to the Internet via a proxy. With no proxy configuration present either in VSCode config nor in environment variables. Attempting to search for an extention nothing happens. No error is displayed. Npm install on the command line works with no problem.

Is there not even a log or something I can view to perhaps get some kind of clue as to what is causing the problem?

P.S. Your documentation for guidance on how to submit a new issue is poor. It suggests checking this 'query' prior to submitting a new bug.
https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc

But this 'query' is just a list of 93 pages of existing issues. Not sure how anyone is expected to trawl through that.

**Comment by Neutrino-Sunset** [comment]
Seen another issue which mentioned 'Help -> Toggle Developer Tools'. Clicking that reveals several ERR_PROXY_CONNECTION_FAILED messages.

Preferences -> Settings -> Http entries are:

  "http.proxy": "",
  "http.proxyStrictSSL": true,
  "http.proxyAuthorization": null,

I have one override configured, which is:

  "http.proxyStrictSSL": false

No proxy environment variables are set at the system level.

**Comment by Neutrino-Sunset** [comment]
I have resolved the problem. It looks like a VS Code bug to me.

I have Firefox installed, it is configured to use no proxy, it works fine.

IE11 is also installed, I never use it. It was configured to use a proxy but no proxy details were entered. It could not connect to the Internet.

I reconfigured IE11 to not use a proxy. Immediately VS Code started working correctly.

The bug is that VS Code has a config file but seems to ignore what's configured there and instead inherits proxy configuration from IE settings.
